recipe/thirdpartyemailpassword/main.go

```diff
@@ -107,8 +107,8 @@ func SendEmailWithContext(input emaildelivery.EmailType, userContext supertokens
 	return (*instance.EmailDelivery.IngredientInterfaceImpl.SendEmail)(input, userContext)
 }
 
-func ThirdPartySignInUp(thirdPartyID string, thirdPartyUserID string, email string) (tpepmodels.SignInUpResponse, error) {
-	return ThirdPartySignInUpWithContext(thirdPartyID, thirdPartyUserID, email, &map[string]interface{}{})
+func ThirdPartyManuallyCreateOrUpdateUser(thirdPartyID string, thirdPartyUserID string, email string) (tpepmodels.ManuallyCreateOrUpdateUserResponse, error) {
+	return ThirdPartyManuallyCreateOrUpdateUserWithContext(thirdPartyID, thirdPartyUserID, email, &map[string]interface{}{})
```

Comment:
Need to add all new functions as well - like FetchThirdPartyConfigWithContext and all

Reply:
Done